[JENKINS-51316] Remove generateKey functionality (#426)
jenkinsci/ec2-plugin: README.md

Access Key which act like a username/password (see IAM section). Because
of the way EC2 works, you also need to have an RSA private key that the
cloud has the other half for, to permit sshing into the instances that

> +are started. Please use the AWS console or any other tool of your choice

Reviewer: extra `+` character?
```suggestion
are started. Please use the AWS console or any other tool of your choice
```

> + to generate the private key to interactively logon to EC2 instances.

Reviewer 2: ```suggestion
+ to generate the private key to interactively log in to EC2 instances.
```

> + to generate the private key to interactively log in to EC2 instances.

Reviewer: ditto
```suggestion
to generate the private key to interactively log in to EC2 instances.
```
